system_tests/validation_mock_test.go

```diff
@@ -278,91 +281,6 @@ func TestValidationServerAPI(t *testing.T) {
 	}
 }
 
-func TestValidationClientRoom(t *testing.T) {
```

Comment:
Why was this test removed? Shouldn't it still work, but just with the new method name?